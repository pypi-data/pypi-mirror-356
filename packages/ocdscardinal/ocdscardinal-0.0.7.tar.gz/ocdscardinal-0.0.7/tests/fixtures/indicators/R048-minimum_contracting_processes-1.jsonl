{"ocid":"F","awards":[{"status":"active","suppliers":[{"id":"F"}],"items":[{"classification":{"id":"10000000"}},{"classification":{"id":"11000000"}}]}]}
{"ocid":"1","awards":[{"status":"active","suppliers":[{"id":"1"}],"items":[{"classification":{"id":"10000000"}}]}]}
{"ocid":"2","awards":[{"status":"active","suppliers":[{"id":"2"}],"items":[{"classification":{"id":"10000000"}}]}]}
{"ocid":"3","awards":[{"status":"active","suppliers":[{"id":"3"}],"items":[{"classification":{"id":"10000000"}}]}]}
{"ocid":"4","awards":[{"status":"active","suppliers":[{"id":"4"}],"items":[{"classification":{"id":"10000000"}}]}]}
{"ocid":"pending award","awards":[{"status":"pending","suppliers":[{"id":"S"}],"items":[{"classification":{"id":"10000000"}},{"classification":{"id":"11000000"}}]}]}
{"ocid":"cancelled award","awards":[{"status":"cancelled","suppliers":[{"id":"S"}],"items":[{"classification":{"id":"10000000"}},{"classification":{"id":"11000000"}}]}]}
{"ocid":"unsuccessful award","awards":[{"status":"unsuccessful","suppliers":[{"id":"S"}],"items":[{"classification":{"id":"10000000"}},{"classification":{"id":"11000000"}}]}]}
{"ocid":"invalid award","awards":[{"status":"invalid","suppliers":[{"id":"S"}],"items":[{"classification":{"id":"10000000"}},{"classification":{"id":"11000000"}}]}]}
{"ocid":"multiple suppliers","awards":[{"status":"active","suppliers":[{"id":"S"},{"id":"T"}],"items":[{"classification":{"id":"10000000"}},{"classification":{"id":"11000000"}}]}]}
{"ocid":"cancelled tender","awards":[{"status":"active","suppliers":[{"id":"F"}],"items":[{"classification":{"id":"10000000"}},{"classification":{"id":"11000000"}}]}],"tender":{"status":"cancelled"}}
