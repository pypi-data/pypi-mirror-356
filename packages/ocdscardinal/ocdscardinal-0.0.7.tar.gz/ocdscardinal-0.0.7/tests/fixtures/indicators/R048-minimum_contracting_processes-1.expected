{"Tenderer":{"F":{"R048":2.0}}}
